ubg/src/mafia/MafiaNight.js

```diff
@@ -85,6 +81,12 @@ export default function MafiaNight({user, usersData, room,
       }
       setPlayers(allPlayers);
     }
+    if (mafiaKill &&
```

Comment:
I was getting an error, it could read uid of undefined, I think maybe it's because it takes a bit to load the redux state, so I added that to check if mafiaKill, doctorSave and detectiveCheck are already defined.

Reply:
couldn't read*

Reply:
In that case maybe add doctorSave and detectiveCheck as well?

Reply:
They are all updated together, but sure.